provider/views.py

```diff
@@ -255,7 +325,36 @@ def error_response(self, request, error, **kwargs):
 
         return self.render_to_response(ctx, **kwargs)
 
+    def get_implicit_response(self, request, client):
+        """
+        Assumes validated and authorized request with a response_type that is
+        associated with an implicit flow.
+        """
+        data = self.get_data(request)
+
+        lookup_kwargs = {
+            "user": request.user,
+            "client": client,
+            "scope": scope.to_int(*data.get('scope', '').split())
```

Comment:
thanks for asking this question @nasthagiri as it revealed a bug in the default scope under implicit flow (will push up a fix + test shortly).

as far as the default scope - no it is not "documented" other than by reading / understanding the code, and observing the convention `constants.SCOPES[0]`.  I'll add a one-liner making that a bit more evident.  Note that we aren't making specific use of the 'scope' parameter for any edx stuff, AFAIK.